.changeset/few-toes-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": major
```

Comment:
If the event notification feature is currently not GA, this can be a minor bump

This PR cannot be merged while this is a major bump

Reply:
It seems this subcommand is not marked as experimental in the subhelp output:

```
$ npx wrangler r2 bucket --help             
wrangler r2 bucket

Manage R2 buckets

COMMANDS
  wrangler r2 bucket create <name>  Create a new R2 bucket
  wrangler r2 bucket update         Update bucket state
  wrangler r2 bucket list           List R2 buckets
  wrangler r2 bucket delete <name>  Delete an R2 bucket
  wrangler r2 bucket sippy          Manage Sippy incremental migration on an R2 bucket
  wrangler r2 bucket notification   Manage event notifications for an R2 bucket

GLOBAL FLAGS
  -j, --experimental-json-config  Experimental: support wrangler.json  [boolean]
  -c, --config                    Path to .toml configuration file  [string]
  -e, --env                       Environment to use for operations and .env files  [string]
  -h, --help                      Show help  [boolean]
  -v, --version                   Show version number  [boolean]
  ```

Can you add `${chalk.hex(betaCmdColor)("[open beta]")}` to the end of the subcommand description please?